### DISCUSSION TITLE
A simple template project for spikingjelly with pytorch-lightning

### DISCUSSION THREAD

**Discussion by Barchid** [fangwei123456/SpikingFlow#150] [Show and tell]
Hello,

I just made a template project to train a Convolutional SNN implemented with SpikingJelly using Pytorch-Lightning. It comes with lots of useful features (tensorboard logging, learning rate finder, etc) without adding tons of boilerplate code. You can use the template project here: [https://github.com/Barchid/spikingjelly-lightning](https://github.com/Barchid/spikingjelly-lightning).

The repository comes with a small example on N-MNIST with a small ConvSNN and a notebook to show the built-in functionalities.

Don't hesitate to poke me if you have suggestions about the template.

**Comment by fangwei123456** [comment]
Thanks for your work!
